internal/cluster/common.go

```diff
@@ -273,6 +273,40 @@ func UpdateMachineCidr(db *gorm.DB, cluster *common.Cluster, machineCidr string)
 			"machine_network_cidr_updated_at": time.Now(),
 		}).Error
 	}
-
 	return nil
 }
+
+func machineNetworksAlreadyExist(cluster *common.Cluster, machineNetworks []string) bool {
```

Comment:
aren't those network related functions? shouldn't they be under network package?

Reply:
It is used by updateMachineNetworks, although it can be moved

Reply:
We even have a dedicated file for storing those, `internal/network/machine_network_cidr.go`, but just a nit, for me okay wherever those live